src/main/java/com/epam/ta/reportportal/core/integration/impl/CreateIntegrationHandlerImpl.java

```diff
@@ -79,6 +83,11 @@ public EntryCreatedRS createGlobalIntegration(IntegrationRQ createRequest, Strin
 		IntegrationService integrationService = integrationServiceMapping.getOrDefault(integrationType.getName(),
 				this.basicIntegrationService
 		);
+
+		ofNullable(createRequest.getName()).map(String::toLowerCase).ifPresent(name -> {
```

Comment:
should we do something if name is null ?

Reply:
validation added